docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/kubernetes.md

```diff
@@ -6,17 +6,19 @@ sidebar_label: Kubernetes (Beta)
 
 <p> <a href="/docs/beta"><span className="beta">Beta</span></a> </p>
```

Comment:
Should that be removed ?

Reply:
Yes, if this is the K8S operator beta->GA